email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -277,9 +276,7 @@ const handleNewMessageEvent = async ({ rowid, params }) => {
     external === true
       ? EXTERNAL_RECIPIENT_ID_SERVER
       : getRecipientIdFromEmailAddressTag(from);
-  const deviceId =
-    external === true ? EXTERNAL_DEVICE_ID_SERVER : senderDeviceId;
-
+  const deviceId = senderDeviceId;
```

Comment:
If you're going to use senderDeviceId directly, better change param name to deviceId